openssl-dynamic/src/main/java/io/netty/internal/tcnative/SSLContext.java

```diff
@@ -469,4 +469,20 @@ public static void setSessionTicketKeys(long ctx, SessionTicketKey[] keys) {
      * @return the mode.
      */
     public static native int getMode(long ctx);
+    
+    /**
+     * Enables OCSP stapling on the given {@link SSLContext}.
+     * 
+     * <p><a href="https://www.openssl.org/docs/man1.0.2/ssl/SSL_set_tlsext_status_type.html">SSL_set_tlsext_status_type</a>
+     * <p><a href="https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html">Search for OCSP</a>
+     */
+    public static native boolean enableOcsp(long ctx, boolean client);
```

Comment:
consider making this throw upon failure, or clarify the return type in javadocs.